src/Forms/FormField.php

```diff
@@ -1583,6 +1583,7 @@ public function getSchemaStateDefaults()
             'value' => $this->Value(),
             'message' => $this->getSchemaMessage(),
             'data' => [],
+            'required' => $this->Required(),
```

Comment:
I don't agree with this... could we discuss why this is here?
you know there's already a validation property for formfields, right?